default/stringtables/de.txt

```diff
@@ -8148,53 +8177,68 @@ BAD_DEFENSE_TROOPS_DESC
 −	Schlechte Boden[[metertype METER_TROOPS]]: 50%
 
 AVERAGE_DEFENSE_TROOPS_DESC
-'''	Mittelmäßige Boden[[metertype METER_TROOPS]]: 100%'''
+⋄	Mittelmäßige Boden[[metertype METER_TROOPS]]: 100%
 
 GOOD_DEFENSE_TROOPS_DESC
-+	Gute Boden[[metertype METER_TROOPS]]: 150%.
++	Gute Boden[[metertype METER_TROOPS]]: 150%
 
 GREAT_DEFENSE_TROOPS_DESC
-++	Hervorragende Boden[[metertype METER_TROOPS]]: 200%.
+++	Hervorragende Boden[[metertype METER_TROOPS]]: 200%
 
 ULTIMATE_DEFENSE_TROOPS_DESC
-+++	Perfekte Boden[[metertype METER_TROOPS]]: 300%.
++++	Perfekte Boden[[metertype METER_TROOPS]]: 300%
 
 BAD_DETECTION_DESC
-−	Schlechte [[metertype METER_DETECTION]]: −20 Malus.
+−	Schlechte [[metertype METER_DETECTION]]: −20 Malus
 
 GOOD_DETECTION_DESC
-+	Gute [[metertype METER_DETECTION]]: +25 Bonus.
++	Gute [[metertype METER_DETECTION]]: +25 Bonus
 
 GREAT_DETECTION_DESC
-++	Hervorragende [[metertype METER_DETECTION]]: +50 Bonus.
+++	Hervorragende [[metertype METER_DETECTION]]: +50 Bonus
 
 ULTIMATE_DETECTION_DESC
-+++	Perfekte [[metertype METER_DETECTION]]: +100 Bonus.
++++	Perfekte [[metertype METER_DETECTION]]: +100 Bonus
 
 BAD_STEALTH_DESC
-−	Schlechte [[metertype METER_STEALTH]]: −20 Malus.
+−	Schlechte [[metertype METER_STEALTH]]: −20 Malus
 
 AVERAGE_STEALTH_DESC
-'''	Mittelmäßige [[metertype METER_STEALTH]].'''
+⋄	Mittelmäßige [[metertype METER_STEALTH]]
 
 GOOD_STEALTH_DESC
-+	Gute [[metertype METER_STEALTH]]: +20 Bonus.
++	Gute [[metertype METER_STEALTH]]: +20 Bonus
 
 GREAT_STEALTH_DESC
-++	Hervorragende [[metertype METER_STEALTH]]: +40 Bonus.
+++	Hervorragende [[metertype METER_STEALTH]]: +40 Bonus
 
 ULTIMATE_STEALTH_DESC
-+++	Perfekte [[metertype METER_STEALTH]]: +60 Bonus.
++++	Perfekte [[metertype METER_STEALTH]]: +60 Bonus
 
 BAD_POPULATION_DESC
 −	Niedrige Bevölkerung 75%
 
 AVERAGE_POPULATION_DESC
-'''	Normale Bevölkerung'''
+⋄	Normale Bevölkerung
 
 GOOD_POPULATION_DESC
 +	Hohe Bevölkerung 125%
 
+GREAT_ASTEROID_INDUSTRY_DESC
++	Guter Asteroidenbergbau: + 0.2 [[metertype METER_INDUSTRY]] je [[metertype METER_POPULATION]] bei Industriefokus in Systemen mit besetzten Asteroidengürteln
+
+LIGHT_SENSITIVE_DESC
+−	Lichtempfindlich: [[metertype METER_POPULATION]] stark reduziert in Sternensystemen der Farbe [[STAR_BLUE]], weniger stark bei Sternen der Farbe [[STAR_WHITE]]
+
+TELEPATHIC_DETECTION_DESC
++	Telepathisches Gespür: kann nahe bevölkerte Welten fühlen
+
+PRECOGNITIVE_DETECTION_DESC
++	Vorwissendes Gespür: kann Planeten jenseits der Sternenstraßen erfühlen
```

Comment:
"Vorausahnende Wahrnehmung: kann Planeten wahrnehmen, die per Sternenstraßen verbunden sind".

Actually I'm wondering if there isn't a better term in German for "starlane" than "Sternenstraße", but I can't think of anything...

Reply:
yeah that one was bugging me too .. but having seen a german youtuber talking about how he play fo for the first time, i think it works .. and t.b.h. havent played much of these games in my native tongue myself since englisch ist so ubiquitös ...

Reply:
taking that verbatim as well

Reply:
> and t.b.h. havent played much of these games in my native tongue myself

Yep, I know what you mean. With games I usually also usually use English, German translation tend to have issues (too long texts, awkward, sometimes even wrong/misleading translations, etc.). English just works better for me... :wink: